data/railsconf/railsconf-2012/videos.yml

```
@@ -321,7 +321,7 @@
   speakers:
     - Bryan Liles
   event_name: RailsConf 2012
-  published_at: "2012-08-22"
+  published_at: "2012-03-23"
```

I assume this should also be April?
```suggestion
  published_at: "2012-04-23"
```